src/species/corn.ts

```diff
@@ -0,0 +1,232 @@
+import * as Populations from '../populations';
+import { IAgent, ISpecies, ITrait } from '../populations-types';
+const { Models: { Agents: { BasicPlant }, Species, Trait } } = Populations;
+
+const maturity = 300;
+
+const cornInfectedTrait: ITrait = new Trait({
+  name: 'infected',
+  possibleValues: [ true, false ],
+  default: false,
+  float: false,
+  mutatable: false
+});
+
+const cornHealthTrait: ITrait = new Trait({
+  name: 'health',
```

Comment:
The Agent class in Populations.js already has a trait named `health`. It's not obvious to me whether attempting to "override" a base class trait in this manner will have the desired effect.